RequestService/RequestService.Repo/Helpers/QuestionExtenstions.cs

```diff
@@ -404,12 +411,15 @@ public static void SetActivityQuestionData(this EntityTypeBuilder<ActivityQuesti
                         entity.HasData(new ActivityQuestions { ActivityId = (int)activity, RequestFormStageId = (int)RequestHelpFormStage.Request, QuestionId = (int)Questions.WillYouCompleteYourself, Location = "pos3", Order = 3, RequestFormVariantId = (int)form, Required = true });
                     }
 
+                    if (form == RequestHelpFormVariant.AgeConnectsCardiff_Public || form == RequestHelpFormVariant.AgeConnectsCardiff_RequestSubmitter)
+                    {
+                        entity.HasData(new ActivityQuestions { ActivityId = (int)activity, RequestFormStageId = (int)RequestHelpFormStage.Request, QuestionId = (int)Questions.RecipientAge, Location = "pos1", Order = 2, RequestFormVariantId = (int)form, Required = false });
```

Comment:
I would have expected Required = true (but haven't checked the ticket to see if it says otherwise).